tests/src/python/test_qgsvectorfilewriter.py

```diff
@@ -567,8 +567,8 @@ def testValueConverter(self):
         self.assertEqual(f['nonconv'], 1)
         self.assertEqual(f['conv_attr'], 'converted_val')
 
-    def testInteger64WriteTabfile(self):
-        """Check writing Integer64 fields to an MapInfo tabfile (which does not support that type)."""
+    def testInteger64WriteUnsupportedFormat(self):
+        """Check writing Integer64 fields to a GMT file (which does not support that type)."""
```

Comment:
joking_mode_enabled: "oh, no, that means that the GMT driver will have to be kept just because of QGIS..." That said, I don't have a more appropriate alternative to suggest, so let's go for it :-)